Modules/Core/Support/SettingsHelper.php

```
@@ -0,0 +1,76 @@
+    {
+        $select = $checked ? 'checked="checked"' : 'selected="selected"';
+
+        // Instant-validate if $value1 is a bool value
+        if (is_bool($value1) && $value2 === null) {
+            echo $value1 ? $select : '';
+            return;
+        }
+
+        switch ($operator) {
+            case '==':
+                $echo_selected = $value1 == $value2;
```

PHPMD (2.15.0)

The variable $echo_selected is not named in camelCase. (undefined)

(CamelCaseVariableName)